### DISCUSSION TITLE
Tdak bisa scan ulang QR

### DISCUSSION THREAD

**Discussion by newfaboc** [kirimwa-id/forum#104] [Tanya Jawab] [answered]
hai mas rio,

mau tanya untuk API WA request apakah ada trobel ? sebab saya coba scan ulang QR gagal.
Tadi pagi sempat bisa jam 07.30 WIB. Namun saya coba lagi pk 08.00 WIB tidak bisa.

Terima kasih

**Comment by rioastamal** [comment, marked as answer]
Mohon maaf, domain expired karena tidak ada notifikasi masuk ke email saya. Saya sedang berupaya untuk mengaktifkan domainnya lagi.

**Comment by newfaboc** [comment]
oh.ok. sudah bisa rio.

Terima kasih.
